unittests/AST/ASTImporterTest.cpp

```diff
@@ -2137,25 +2134,26 @@ TEST_F(ImportFriendFunctions, Lookup) {
   auto FromD = FirstDeclMatcher<FunctionDecl>().match(FromTU, Pattern);
   ASSERT_TRUE(FromD->isInIdentifierNamespace(Decl::IDNS_OrdinaryFriend));
   ASSERT_TRUE(!FromD->isInIdentifierNamespace(Decl::IDNS_Ordinary));
-  auto Name = FromD->getDeclName();
+  auto FromName = FromD->getDeclName();
   {
     CXXRecordDecl *Class =
         FirstDeclMatcher<CXXRecordDecl>().match(FromTU, cxxRecordDecl());
-    auto lookup_res = Class->noload_lookup(Name);
+    auto lookup_res = Class->noload_lookup(FromName);
     ASSERT_EQ(lookup_res.size(), 0u);
-    lookup_res = cast<TranslationUnitDecl>(FromTU)->noload_lookup(Name);
+    lookup_res = cast<TranslationUnitDecl>(FromTU)->noload_lookup(FromName);
     ASSERT_EQ(lookup_res.size(), 1u);
   }
 
-  Import(FromD, Lang_CXX);
+  auto ToD = cast<FunctionDecl>(Import(FromD, Lang_CXX));
+  auto ToName = ToD->getDeclName();
 
   {
     Decl *ToTU = ToAST->getASTContext().getTranslationUnitDecl();
     CXXRecordDecl *Class =
         FirstDeclMatcher<CXXRecordDecl>().match(ToTU, cxxRecordDecl());
-    auto lookup_res = Class->noload_lookup(Name);
+    auto lookup_res = Class->noload_lookup(ToName);
     EXPECT_EQ(lookup_res.size(), 0u);
-    lookup_res = cast<TranslationUnitDecl>(FromTU)->noload_lookup(Name);
+    lookup_res = cast<TranslationUnitDecl>(FromTU)->noload_lookup(FromName);
```

Comment:
Probably this test should be the on "To" side not "From"? (The question applies to all such later cases.)

Reply:
Yes that is right, it should be `ToTU`.